swaybar/swaybar-protocol.7.scd

```diff
@@ -8,10 +8,10 @@ swaybar-protocol - JSON status line protocol for swaybar
 
 swaybar defines the following JSON protocol to specify what information is
 displayed in the status line on the right side of swaybar. The protocol
-comprises a header, that is a JSON object, followed by an infinite JSON
-array that represents the information to display. All communication is done by
-writing the status line to standard output and reading events from standard
-input.
+comprises a header, that is a JSON object, followed by a newline (*0x0A*),
```

Comment:
You might want to replace 'a newline' with 'an ASCII newline character'.

Reply:
The current wording sounds explicit enough for me, especially since 0x0A is mentioned.